pretyping/reductionops.mli

```diff
@@ -182,6 +182,8 @@ val whd_betalet_stack : stack_reduction_function
 
 (** {6 Head normal forms } *)
 
+val whd_const_stack : Constant.t -> stack_reduction_function
+val whd_const : Constant.t -> reduction_function
```

Comment:
This is a very specific function, I don't think you should export the `whd_const_stack` variant.